staging/src/k8s.io/apimachinery/pkg/util/sets/doc.go

```diff
@@ -17,4 +17,4 @@ limitations under the License.
 // Code generated by set-gen. DO NOT EDIT.
```

Comment:
You need to change the code generator :)

Reply:
Done. Thank you!!